src/components/atoms/Icon/icons.ts

```diff
@@ -104,6 +104,8 @@ import Scale from "./Icons/scale.svg";
 import MapIcon from "./Icons/map.svg";
 import NoProjects from "./Icons/noProjects.svg";
 import TopPage from "./Icons/topPage.svg";
+import Moon from "./Icons/moon.svg";
+import Sun from "./Icons/sun.svg";
```

Comment:
Any chance we can name them more meaningfully? Like lightTheme.svg and darkTheme.svg?

Reply:
Hmmm, in my opinion I think the names should describe the content as it is, so If someone in the future needs to use the "sun" or "moon" icons, he knows they exist.
But  "lightTheme" and "darkTheme" as a name, does not provide meaningful description of the content of the image.
What do you think?